tests/test_invoices.py

```diff
@@ -533,6 +533,19 @@ def test_waitanyinvoice(node_factory, executor):
         l2.rpc.waitanyinvoice('non-number')
 
 
+def test_signinvoice(node_factory, executor):
+    # Setup
```

Comment:
```suggestion
    """ Make sure that the signinvoice work as expected while re-sign a previous invoice """
    # Setup
```

This is a personal preference, that I find useful in case of refactoring the test in the future